### DISCUSSION TITLE
Unable to upload archive

### DISCUSSION THREAD

**Discussion by TheCarpetMerchant** [codemagic-ci-cd/codemagic-docs#1936] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

I'm getting this error after enabling code signing. I'm building in release mode, with a development profile, and I checked "Submit to TestFlight beta review".

```
Upload "/Users/builder/clone/build/ios/ipa/ao3reader.ipa" to App Store Connect
{
    "tool-version": "6.315.14315",
    "tool-path": "/Applications/Xcode-14.3.app/Contents/SharedFrameworks/ContentDeliveryServices.framework/Versions/A/Frameworks/AppStoreService.framework",
    "os-version": "13.3.1",
    "product-errors": [
        {
            "message": "Unable to upload archive.",
            "userInfo": {
                "NSLocalizedDescription": "Unable to upload archive.",
                "NSLocalizedFailureReason": "Failed to authenticate for session: (\n    \"Error Domain=ITunesConnectionAuthenticationErrorDomain Code=-26000 \\\"Failure to authenticate.\\\" UserInfo={NSLocalizedRecoverySuggestion=Failure to authenticate., NSLocalizedDescription=Failure to authenticate., NSLocalizedFailureReason=App Store operation failed.}\"\n)"
            },
            "code": -1011
        }
    ]
}
Failed to upload archive at "/Users/builder/clone/build/ios/ipa/ao3reader.ipa"
Failed to publish /Users/builder/clone/build/ios/ipa/ao3reader.ipa

Failed to publish ao3reader.ipa to App Store Connect.

Build failed :|


Publishing failed :|
Failed to publish ao3reader.ipa to App Store Connect.
```

### Build id

64c9366f2e3d4896d0a11a22

**Comment by TheCarpetMerchant** [comment]
Looking at the full log :
```
> app-store-connect publish --path /Users/builder/clone/build/ios/ipa/ao3reader.ipa --key-id 264H8KGZXD --issuer-id 'https://labs.codemagic.io/your-first-flutter-app-to-appstore/#4' --private-key @env:APP_STORE_CONNECT_PUBLISHER_PRIVATE_KEY
```

No wonder I wasn't finding anything on this error lol.

**Comment by SahanNagodaDigital400** [comment]
Have you find a solution @TheCarpetMerchant ?
